### ISSUE TITLE
abgelaufen

### ISSUE THREAD

**Issue by Karaoglan1980** [dougy147/mcbash#8]
Hello, tried a few times and it just expired.  Can someone try this for me and see if it works for you?  http://de.caneriptv.com:8080 Thank you

**Comment by ian10951** [comment]
Yes its still failing on several Gen 1 panels and as yet I've been unable to determine why. It seems all Gen 2 are OK though.

**Comment by Karaoglan1980** [comment]
Could you please test a URL you have and send me this result so I can try it and see if I get the same result?
